povm_toolbox/library/locally_biased_classical_shadows.py

```diff
@@ -8,61 +8,75 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""TODO."""
+"""LocallyBiasedClassicalShadows."""
 
 from __future__ import annotations
 
 import numpy as np
 from numpy.random import Generator
 
-from .randomized_projective_measurements import RandomizedProjectiveMeasurements
+from .mutually_unbiased_bases_measurements import MutuallyUnbiasedBasesMeasurements
 
 
-class LocallyBiasedClassicalShadows(RandomizedProjectiveMeasurements):
-    """A locally-biased classical shadows POVM."""
+class LocallyBiasedClassicalShadows(MutuallyUnbiasedBasesMeasurements):
+    """A locally-biased classical shadows POVM.
```

Comment:
Nice, good idea to make it a subclass of `MutuallyUnbiasedBasesMeasurementa` now that we have it!